test/check-machines-create

```diff
@@ -96,10 +96,6 @@ class TestMachinesCreate(VirtualMachinesCase):
         # Memory unit conversion
         runner.unitConversionTest(TestMachinesCreate.VmDialog(self, sourceType='disk-image'))
 
-        # try to CREATE WITH DIALOG ERROR
-        # name
-        runner.checkDialogFormValidationTest(TestMachinesCreate.VmDialog(self, "", storage_size=1), {"vm-name": "Name must not be empty"})
```

Comment:
This should be replaced with the new expected behaviour of auto-creating a name and ensuring that VM creation actually works and has a name of the expected form.